operations/sthree.go

```diff
@@ -531,7 +531,7 @@ func setVerboseLogging(verbose bool) error {
 		info := sender.Level()
 		info.Threshold = level.Debug
 
-		return errors.Wrap(sender.SetLevel(info), "problem setting logger threshold to debug")
+		return errors.Wrap(sender.SetLevel(info), "setting logger threshold to debug")
```

Comment:
`to info`

Reply:
It's setting the debug threshold on the level info.